.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Notes:
+1. Only post _bug reports_ here.
+2. Use the appropriate template for _feature requests_.
+3. Please direct questions to the [`#formulas` channel on Slack](https://saltstackcommunity.slack.com/messages/C7LG8SV54/), which is bridged to `#saltstack-formulas` on Freenode.
+-->
+
+#### Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+
+
+#### Setup
+<!-- Provide links to the SLS files and/or relevant configs (be sure to remove sensitive info). -->
+
+
+
+#### Steps to reproduce the bug
+<!-- Include debug logs if possible and relevant. -->
```

Comment:
Can be interesting to explain how to 'debug log', like running salt with option `-l debug`

Reply:
but if we consider user is using Kitchen images, than it's already in debug mode _(i think)_

Reply:
Good idea.

Reply:
We're currently using `info` rather than `debug`, so I've taken the suggestion and added it.